### DISCUSSION TITLE
How do I change the auto-complete accept from `<CR>` to `<Tab>`?

### DISCUSSION THREAD

**Discussion by jsatk** [LazyVim/LazyVim#5181] [Q&A]
Basically the title. Having `<CR>` be the confirm key really throws things off.

An example, if I try to type the following (assume the `<CR>` is me pressing it, not typing it) and the LSP suggests `fooBar` when I press `<CR>` instead of going to a new line it will autocomplete to `fooBar`.

```
foo<CR>
bad
```

**Comment by dpetka2001** [comment]
That depends on the completion engine you're using. Assuming you're using the default blink, there are 3 presets as described in their [docs](https://github.com/Saghen/blink.cmp?tab=readme-ov-file#configuration) (`default`, `enter` and `super-tab`). Choose one of the 3 that is more comfortable for you or pass entirely new mappings of your own.
